weblogic-azure-aks/src/main/arm/scripts/common.sh

```diff
@@ -10,8 +10,10 @@ export constAdminServerName='admin-server'
 export constClusterName='cluster-1'
 export constClusterT3AddressEnvName="T3_TUNNELING_CLUSTER_ADDRESS"
 export constDefaultJavaOptions="-Dlog4j2.formatMsgNoLookups=true -Dweblogic.StdoutDebugEnabled=false" # the java options will be applied to the cluster
+export constDefaultJVMArgs="-Djava.security.egd=file:/dev/./urandom -XX:MinRAMPercentage=25.0 -XX:MaxRAMPercentage=50.0 " # the JVM options will be applied to the cluster
```

Comment:
Are we sure we want to do this for **all** wls versions? The other script only applies to 14c, but this change applies unconditionally. Is that what we want?

Reply:
Yes, you are right, we should specify `-Xms256m -Xmx512m` for lower version. Thanks for the finding.